### PR TITLE
Remove contact from MD when contact is deleted

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

File: geocat/src/main/java/org/fao/geonet/geocat/services/reusable/Reject.java [hunk 1]
```diff
@@ -83,6 +91,9 @@ public Element exec(Element params, ServiceContext context) throws Exception {
         boolean testing = Boolean.parseBoolean(Util.getParam(params, "testing", "false"));
         boolean isValidObject = Boolean.parseBoolean(Util.getParam(params, "isValidObject", "false"));
 
+        this.dataManager = context.getBean(DataManager.class);
```
Comment (pmauduit) [inline geocat/src/main/java/org/fao/geonet/geocat/services/reusable/Reject.java]: `@Autowired` ?

[edit] maybe not, actually, I cannot remember if a jeeves service is also a bean managed by spring.

Comment (Vampouille) [inline geocat/src/main/java/org/fao/geonet/geocat/services/reusable/Reject.java]: for me it is not spring managed :-(

**Comment by Vampouille** [comment]
Work in progress, please do not merge.

**Comment by Vampouille** [comment]
6517d91 should be removed because "delete" action on validated contacts should reject it.

**Pull request closed without merging**